ui/src/message/Message.tsx

```diff
@@ -1,4 +1,4 @@
-import {Button, Theme, useMediaQuery, useTheme} from '@mui/material';
+import {Box, Button, Theme, useMediaQuery, useTheme} from '@mui/material';
```

Comment:
I've updated the description with new screenshots. The mobile style now has a different header layout. @eternal-flame-AD do you have a better idea for the application image in the mobile layout, it looks a little out of place but at least it's more compact?